tests/json-fixtures/test_transactions.py

```diff
@@ -80,12 +78,15 @@ def test_transaction_fixtures(fixture_name, fixture):
 
         sender = to_canonical_address(fixture['sender'])
 
-        assert transaction.sender == sender
+        try:
+            assert transaction.sender == sender
+        except BadSignature:
+            assert not (27 <= transaction.v <= 34)
     else:
         # check RLP correctness
         try:
             transaction = rlp.decode(fixture['rlp'], sedes=TransactionClass)
-        except rlp.exceptions.ObjectDeserializationError:
+        except (rlp.exceptions.ObjectDeserializationError, rlp.exceptions.DecodingError, TypeError):
```

Comment:
I'm a little worried that the tests passing are because this is just catching *all* of the errors.  I'm traveling today and don't have time to check this, but can you verify that the errors being caught here are appropriate error conditions by potentially logging out how many of the tests succeed because of the subsequent `return` statement.